tests/unit/src/test/scala/tests/WorkspaceSymbolLspSuite.scala

```diff
@@ -142,10 +142,13 @@ class WorkspaceSymbolLspSuite extends BaseLspSuite("workspace-symbol") {
       _ = server.workspaceSymbol("scala.None")
       option = ".metals/readonly/scala/Option.scala"
       _ <- server.didOpen(option)
-      references <- server.references(option, "object None")
+      references = server.references(option, "object None")
       _ = assertNoDiff(
         references,
-        """|a/src/main/scala/a/A.scala:4:24: info: reference
+        """|.metals/readonly/scala/Option.scala:527:13: info: reference
```

Comment:
I think this change is a regression. It seems we do include declaration by default now?

Reply:
As described in PR:
Here (https://github.com/scalameta/metals/pull/1388/files#diff-2c60d9818336621db1490d63594edff5R151) I had to add declaration to expected results, because in tests server.references(...) creates underneath new ReferenceContext(true) (where true is for includeDeclaration) which indicates declaration should be added to results.